judge/management/commands/runmoss.py

```diff
@@ -0,0 +1,46 @@
+from django.core.management.base import BaseCommand
+
+from judge.models import *
+
+from moss import *
+
+
+class Command(BaseCommand):
+    help = 'checks for duplicate code using MOSS'
+
+    lang_mapping = {
+        ('C++', MOSS_LANG_CC),
+        ('C', MOSS_LANG_C),
+        ('Java', MOSS_LANG_JAVA),
+        ('Python', MOSS_LANG_PYTHON),
+    }
+
+    def add_arguments(self, parser):
+        parser.add_argument('contest', help='the id of the contest')
+        parser.add_argument('key', help='MOSS API key')
+
+    def handle(self, *args, **options):
+        contest = options['name']
+        moss_key = options['key']
+
+        for problem in Contest.objects.get(key=contest).problems.order_by('code').all():
+            print '========== %s / %s ==========' % (problem.code, problem.name)
+            for dmoj_lang, moss_lang in lang_mapping:
+                print "%s: " % dmoj_lang,
+                subs = Submission.objects.filter(contest__participation__contest__key=contest,
```

Comment:
Should we filter `virtual=0` and perhaps `virtual=-1`? I don't think post contest stuff should be what we are trying to catch with MOSS on a contest.